pallets/elections/src/lib.rs

```diff
@@ -48,7 +51,8 @@ impl<AccountId> Default for EraValidators<AccountId> {
     }
 }
 
-#[derive(Decode, Encode, TypeInfo, Debug, Clone, Copy, PartialEq, Eq, Default)]
+#[derive(Decode, Encode, TypeInfo, Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
we can move this to primitives to avoid adding `serde` dependency to the pallet

Reply:
the other alternative is to take just `reserved_seats: u32` and `non_reserved_seats: u32` in `GenesisConfig`; this has such an advantage that `CommitteeSeats` would stay in pallet where it belongs; wdyt?

Reply:
I would move it to primitives anyway, an additional reason is that this struct is used in other places as well (e2e tests, chain_spec.rs etc)

Reply:
done